nixos/modules/config/users-groups.nix

```diff
@@ -685,7 +685,7 @@ in {
       shadow.gid = ids.gids.shadow;
     };
 
-    system.activationScripts.users = {
+    system.activationScripts.users = if !config.systemd.sysusers.enable then {
```

Comment:
Because I need the `else` clause. The activationScript still needs to exist (as an empty string) otherwise other activationScripts that depend on e.g. `users` might fail.

Reply:
Should be lib.optionalString + a comment about why then.